src/activities/Elsa.Activities.Kafka/Models/MessageReceivedInput.cs

```diff
@@ -8,6 +8,8 @@ namespace Elsa.Activities.Kafka.Models
 {
     public class MessageReceivedInput
     {
+        public string? MessageHeaders { get; set; }
```

Comment:
Wouldn't it be more convenient to get the headers as a collection of headers instead of a serialized JSON string? Otherwise, the workflow activities first need to deserialize this, which also means they need to know what sort of JSON format to expect, like PascalCase, camelCase, etc.

Reply:
Sure, I'll try

Reply:
@sfmskywalker  I've changed it to Dictionary<string, string>.